<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetupCanvas</class>
 <widget class="QDialog" name="SetupCanvas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="displayInfoTable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>281</width>
     <height>281</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string>Name</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>X</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Y</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Width</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Height</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Refresh rate</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>DPI</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Depth</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>Details</string>
    </property>
   </column>
  </widget>
  <widget class="QListWidget" name="videoAdapterlist">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>41</y>
     <width>281</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="testScreenButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>40</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Test Screen(s)</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="selectOutputBox">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>30</y>
     <width>171</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>Select Output</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>151</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>530</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>530</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="canvasXlabel">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>90</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Canvas X:</string>
   </property>
  </widget>
  <widget class="QLabel" name="canvasYlabel">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>130</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Canvas Y:</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="canvaspreviewBox">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>280</y>
     <width>471</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string>Canvas Preview</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
